tools/mage/deploy/deploy.go

```diff
@@ -591,3 +586,30 @@ func customResourceVersion() string {
 	// and also the same format used by the master stack.
 	return fmt.Sprintf("%s (%s)", util.Semver(), util.CommitSha())
 }
+
+// Takes a string  and returns the panther- prefixed, lowercased slice of words(strings) (separated by spaces).
+// e.g "oRg-ApI" -> []string{"panther-org-api"}
+// e.g "one two THREE" -> []string{"panther-one", "panther-two", "panther-three"}
+func PantherNames(setString string) []string {
+	set := strings.Fields(setString)
+	for i, entry := range set {
+		entry = strings.ToLower(entry)
+		if !strings.HasPrefix(entry, "panther-") {
+			entry = "panther-" + entry
+		}
+		set[i] = entry
+	}
+	return set
+}
+
+// Takes a type, set of types, and a function to call with each type
+func RunSet(label string, nameset string, callFn func(string) error) int {
+	pantherNameSet := PantherNames(nameset)
+	for _, entry := range PantherNames(nameset) {
+		log.Info(label, ": ", entry)
```

Comment:
Are you sure this works as intended? I think you want

```suggestion
		log.Infof("%s: %s", label, entry)
```

(sorry I missed that the first time)